testing/output-only-actors.md

```diff
@@ -98,7 +102,11 @@ That's a nice chunk of code, let's break it down and focus on the important bits
 
 ```
 
-Remember, we are attempting to verify that when we print to `MyImportantClass`, we get the correct output on the stream. In a real world example, our class would probably be doing some sort of formatting and wouldn't just be a pass through of the data. Instead of testing file stream directly, we are testing a stub `_TestStream` that is standing in a standard library `OutStream` interface. In real code, this would probably be the concrete actor `FileStream` or similar. Our stub implements the OutStream interface and records everything we write to it:
+Note the use of `TestHelper.long_test(1_000_000_000)`, where we tell the test framework that our test will continue
```

Comment:
It would be better to follow [formatting guidelines][1] and do not hard-wrap paragraph.

[1]: https://github.com/ponylang/pony-patterns/blob/master/CONTRIBUTING.md#how-to-format-your-pattern